src/components/AuctionBlock.tsx

```diff
@@ -343,13 +343,11 @@ const InfoCol = styled.div`
 
 const InfoLabel = styled.div`
     color: ${(props) => props.theme.colors.secondary};
-    font-size: ${(props) => props.theme.font.extraSmall};
```

Comment:
Rather than delete all these `font-size` items, can we instead just create the missing size? This will make things easier when we start to normalize the font sizing across the entire app.